azure/az-cloud-context/functions/public/get-currentazcloudcontext.ps1

```diff
@@ -0,0 +1,10 @@
+Function Get-CurrentAzCloudContext
+{
```

Comment:
Ah, so $global is the static thingy in PS - cool

Reply:
> Ah, so $global is the static thingy in PS - cool

Not sure if global is the way to go, but one option for us to use.  That way we can easily switch the context across everything we are doing in those particular modules.  If you have a better idea of architecting this, I'm all ears.